tasks/pull.yaml

```diff
@@ -32,3 +32,31 @@ tasks:
             CURRENT_PKG_NAME=$(cat zarf.yaml | yq .metadata.name)
             test -f zarf-package-${CURRENT_PKG_NAME}-${UDS_ARCH}-${CURRENT_VERSION}.tar.zst || mv zarf-package-${CURRENT_PKG_NAME}-${UDS_ARCH}-*.tar.zst zarf-package-${CURRENT_PKG_NAME}-${UDS_ARCH}-${CURRENT_VERSION}.tar.zst
           fi
+
+  - name: latest-bundle-release
+    inputs:
+      path:
+        description: Path relative to the repositories root where the bundle things happen
+        default: .
+      spoof_release:
+        description: Whether to spoof the pulled bundle version to the current repo version
+        default: "false"
+    actions:
+      - description: Get the current uds bundle name
+        cmd: cat ${{ .inputs.path }}/uds-bundle.yaml | yq .metadata.name
+        setVariables:
+          - name: BUNDLE_NAME
+      - description: Get latest tag version from OCI
+        cmd: ./uds zarf tools registry ls ${TARGET_REPO}/bundles/${BUNDLE_NAME} | sort -V | tail -1
+        setVariables:
+          - name: LATEST_VERSION
+      - description: Pull the latest bundle release
+        cmd: ./uds pull oci://${TARGET_REPO}/bundles/${BUNDLE_NAME}:${LATEST_VERSION} --no-progress -o ${{ .inputs.path }}
+      # TODO (@WSTARR): This is currently needed to get around the chicken+egg condition when release please updates the version in GH
+      - description: Spoof the latest release to the current version
+        cmd: |
+          if [ ${{ .inputs.spoof_release }} != "false" ]; then
+            CURRENT_VERSION=$(cat ${{ .inputs.path }}/uds-bundle.yaml | yq .metadata.version)
+            CURRENT_PKG_NAME=$(cat ${{ .inputs.path }}/uds-bundle.yaml | yq .metadata.name)
```

Comment:
do we want to use path both for where the bundle is going, and where the definition is?  Do we want to normalize the package part as well?

Reply:
For the first part, yes or at least that's how it is in every repo using bundles today. For the second part are you saying because the var is named PKG or normalize package to go to the same location as the zarf.yaml?

Reply:
i made the other changes. dont think we should change this.

Reply:
fixed